io.github.zzzgydi.ClashVerge.yml

```diff
@@ -0,0 +1,95 @@
+app-id: io.github.zzzgydi.ClashVerge
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: clash-verge
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin:/usr/lib/sdk/rust-stable/bin
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.38.6.tar.xz
+        sha256: 1c614c9589389db1a79ea9ba4293bbe8ac3ab0a2234cac700935fae0724ad48b
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: <a href="webkitgtk\-(2\.38\.\d+)\.tar\.xz">
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORT=GTK
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+      - name: bubblewrap
```

Comment:
Why do you need this? Flatpak-spawn is available, and it should be used; bubblewrap will fail to work inside of the flatpak sandbox otherwise.

Reply:
Because bubblewrap is the dependency of webkit2gtk

Reply:
Webkitgtk should be using flatpak-spawn, and as far as I'm aware, it uses it when running in Flatpak. Not even the Epiphany Flatpak ships it, which, as far as I'm aware, uses webkitgtk. 

Bubblewrap is unnecessary, and again, *will not work*.